test/e2e/e2e_test.go

```diff
@@ -41,7 +44,25 @@ import (
 )
 
 func init() {
-	framework.ViperizeFlags()
+	// Register framework flags, then handle flags and Viper config.
+	framework.HandleFlags()
+	viperconfig.ViperizeFlags()
+	framework.AfterReadingAllFlags(&framework.TestContext)
+
+	// TODO: Deprecating repo-root over time... instead just use gobindata_util.go , see #23987.
```

Comment:
Please open an issue on this, it's a thorny beast.

Reply:
Do we need a new one or can we reopen one of the existing ones? This TODO has been in the code for a while already and previous attempts to get this resolved have stalled:

https://github.com/kubernetes/kubernetes/issues/23987
https://github.com/kubernetes/kubernetes/issues/31270